crates/ruff_notebook/src/notebook.rs

```diff
@@ -149,7 +149,7 @@ impl Notebook {
     {
         let trailing_newline = reader.seek(SeekFrom::End(-1)).is_ok_and(|_| {
             let mut buf = [0; 1];
-            reader.read_exact(&mut buf).is_ok_and(|_| buf[0] == b'\n')
+            reader.read_exact(&mut buf).is_ok_and(|()| buf[0] == b'\n')
```

Comment:
Not convinced by that clippy change tbh (the other ones are good though)

Reply:
I don't mind disabling that lint... Here's the documentation of the rule and why https://rust-lang.github.io/rust-clippy/master/index.html#/ignored_unit_patterns

I was like, meh but then it's rare and not too bad

Reply:
I'm fine with it, don't feel strongly.